docs/build.md

```diff
@@ -0,0 +1,66 @@
+# Electron Build Instructions
+
+## Common
+
```

Comment:
Maybe add a "warning" to communicate that this process will produce an unsigned app installer which is meant to be a testing version only. The final installer should be signed.